### PR TITLE
[ARP] working agreement: 2 reviewers per PR

### PR DESCRIPTION
**Pull request by ameowlia**
Please leave this PR open until the ARP working group discusses and votes.

### CODE REVIEW

File: toc/working-groups/002-PRs-Require-2-Reviewers.md
Comment (geofffranks) [inline toc/working-groups/002-PRs-Require-2-Reviewers.md:13]: +1 in comments, or can we just use the built-in review/approve features of github? I

File: toc/working-groups/002-PRs-Require-2-Reviewers.md
Comment (loewenstein) [reply toc/working-groups/002-PRs-Require-2-Reviewers.md:11]: Looking at what e.g. the Istio community does, we could also allow `lgtm` votes from working group members and require an approval from an approver. That way feedback from multiple people would be required, but only one had to be an approver.

Comment (ameowlia) [reply toc/working-groups/002-PRs-Require-2-Reviewers.md:11]: If we use github reviews you are not allowed to review your own PR. Let's stick to that since that is how the tool works, plus it means no one can merge their own code with no review.

**Comment by rkoster** [comment]
How are y’all handeling PRs submitted by a pair?  As in all commits have 2 names attached to them?

**Review by loewenstein — CHANGES_REQUESTED** [review]
The location for these working agreements seems to be off. Did you indent to introduce something like `toc/working-groups/app-runtime-platform/` and collect WG specific content?

**Pull request closed without merging**
